### PR TITLE
Gradle upgrade 6.4.1

### PR DESCRIPTION
**Pull request by KazankovMarch**
`gradle wrapper --gradle-version 6.4.1` with no warnings

### CODE REVIEW

File: aggregating-profiler-jmh/build.gradle.kts [hunk 1]
```diff
@@ -19,16 +19,16 @@ dependencies {
...
+    implementation(Libs.slf4j_api)
+    implementation(Libs.kotlin_jdk8)
+    implementation(Libs.kotlin_stdlib)
+    implementation(Libs.kotlin_reflect)
+    implementation(Libs.slf4j_simple)
+    implementation(Libs.jmh)
+    implementation(Libs.jmhGeneratorAnn)
+    implementation(Libs.jmhGeneratorBytecode)
+
+    implementation(project(":aggregating-profiler"))
```
Comment (KazankovMarch) [inline aggregating-profiler-jmh/build.gradle.kts:31]: as far as I understand, JMH module is used to test profiler's perfomance, so it shouldn't be imported as a dependency. Therefore i used only `implementation` configuration here

File: aggregating-profiler-prometheus/build.gradle.kts [hunk 2]
```diff
@@ -10,15 +7,14 @@ dependencies {
     /**
      * Runtime
      */
-    compile(Libs.slf4j_api)
-
-    compile(project(":aggregating-profiler"))
+    api(Libs.slf4j_api)
```
Comment (TimurKasatkin) [inline aggregating-profiler-prometheus/build.gradle.kts]: image
Actually, I don't see any usages of logging libraries in aggregating-profiler-prometheus

File: aggregating-profiler-prometheus/build.gradle.kts [hunk 3]
```diff
@@ -10,15 +7,14 @@ dependencies {
...
+    api(project(":aggregating-profiler"))
+    api(Libs.kotlin_logging)
```
Comment (TimurKasatkin) [inline aggregating-profiler-prometheus/build.gradle.kts]: same question about logging library necessity

**Review by TimurKasatkin — CHANGES_REQUESTED** [review]

File: aggregating-profiler-prometheus/build.gradle.kts (hunk 2 above)
Comment (KazankovMarch) [reply aggregating-profiler-prometheus/build.gradle.kts]: I removed that dependency

File: aggregating-profiler-prometheus/build.gradle.kts (hunk 3 above)
Comment (KazankovMarch) [reply aggregating-profiler-prometheus/build.gradle.kts]: removed dependency from api configuration (but not from testImplementation)

**Review by TimurKasatkin — APPROVED** [review]

**Pull request merged**